scripts/nrf_provision/fast_pair/README.rst

```diff
@@ -10,7 +10,7 @@ Fast Pair provision script
 This Python script generates a hexadecimal file required for the :ref:`Google Fast Pair integration in the nRF Connect SDK <ug_bt_fast_pair>`.
 
 .. note::
-   When you :ref:`use Google Fast Pair with the nRF Connect SDK <ug_bt_fast_pair>`, the script is automatically called by the build system.
+   When you :ref:`use Google Fast Pair with the nRF Connect SDK <ug_bt_fast_pair>` and enable the ``SB_CONFIG_BT_FAST_PAIR`` Kconfig option, the script is automatically called by the build system.
```

Comment:
I would not mention only the Sysbuild option here (you can build with Fast Pair also without Sysbuild). Describing how it works under the hood in the user guide and linking there from other places may be a good idea (you can also follow this approach for samples by just mentioning that Fast Pair samples' configuration allows to generate provisioning data when building with and without Sysbuild)

Reply:
ok